src/redux/actions/index.js

```diff
@@ -66,4 +66,42 @@ export const fetchProjects = (userid) => {
     }
 }
 
+//Get tasks actions
+//Get project actions
+export const fetchTasksRequest = () => {
+    return {
+        type: 'FETCH_TASKS_REQUEST'
+    }
+}
+
+export const fetchTasksSuccess = (tasks) => {
+    return {
+        type: 'FETCH_TASKS_SUCCESS',
+        payload: tasks,
+    }
+}
+
+export const fetchTasksFailure = (error) => {
+    return {
+        type: 'FETCH_TASKS_FAILURE',
+        payload: error,
+    }
+}
 
+export const fetchTasks = (userid) => {
+    return async dispatch => {
+        dispatch(fetchTasksRequest())
+        
+        let tasks = []
+        Fire.firestore().collection('tasks').where("userId", "==", userid).get()
```

Comment:
Je data access is nu verspreid over 2 plekken, als je nu firebase er uit mieteetd of de tabellen of de queries aanpast ben je de sjaak kijk of je het kan centraliseren